chia/wallet/wallet_state_manager.py

```diff
@@ -1045,7 +1045,6 @@ async def add_coin_states(
                     # This only succeeds if we don't raise out of the transaction
                     await self.retry_store.remove_state(coin_state)
 
-                    existing: Optional[WalletCoinRecord]
```

Comment:
it might be nice to preserve the hint though, down where we assign to this variable